config/jobs/kubernetes/sig-arch/kubernetes-depstat.yaml

```diff
@@ -5,6 +5,7 @@ presubmits:
     path_alias: k8s.io/kubernetes
     always_run: false
     optional: true
+    run_if_changed: '/go.(mod|sum)|vendor/'
```

Comment:
did you see the slashes anywhere else @RinkiyaKeDad?

Can we keep it simple?
```
run_if_changed: '^(go.mod|go.sum|vendor)'
```

Reply:
> did you see the slashes anywhere else @RinkiyaKeDad?

I saw a trailing slash [here](https://github.com/kubernetes/test-infra/blob/master/config/jobs/image-pushing/k8s-staging-ingress-controller-conformance.yaml#L8) and at a couple of other places so I thought I should keep them both 😅 

Will update! :)